src/routes/resources/resources.routes.ts

```
@@ -42,6 +62,10 @@ export const create = createRoute({
       z.object({ message: z.string(), success: z.boolean().default(false) }),
       "Bad Request"
     ),
+    [HttpStatusCodes.INTERNAL_SERVER_ERROR]: jsonContent(
+      z.object({ message: z.string(), success: z.boolean().default(false) }),
+      "Bad Request"
+    ),
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)